scripts/version updates/update_meilisearch_to_specific_version.sh

```diff
@@ -0,0 +1,141 @@
+## DUMPS
+
+current_meilisearch_version=$(curl -X GET 'http://localhost:7700/version' --header "X-Meili-API-Key: $MEILISEARCH_MASTER_KEY" -s)
+
+echo "Current meilisearch version $current_meilisearch_version \n"
+
+if [ $# -eq 0 ]
+then
+  echo "MeiliSearch Version not provided \n"
+  exit
+fi
+
+## Dump just in case not compatible
+echo "Creation of a dump in case new version does not have compatibility with the current MeiliSearch "
+dump_return=$(curl -X POST 'http://localhost:7700/dumps' --header "X-Meili-API-Key: $MEILISEARCH_MASTER_KEY" -s)
+
+status_code=$?
+if  [ $status_code -ne 0 ]
+then 
+  echo "Could not create dump: $dump_return"
+  exit
+fi
+
+dumpid=$(echo $dump_return | cut -d '"' -f 4)
+
+echo  "\n"
+
+until curl -X GET "http://localhost:7700/dumps/$dumpid/status"  --header "X-Meili-API-Key: $MEILISEARCH_MASTER_KEY" -s | grep '"status":"done"'
+do
+  echo "MeiliSearch is still creating the dump \n"
+  sleep 2
+done
+
+echo "MeiliSearch finished creating the dump \n"
```

Comment:
```suggestion
echo "MeiliSearch finished creating the dump $dumpid\n"
```
Maybe it could be nice to add the id of the dump